db/schema.rb

```diff
@@ -662,6 +663,13 @@
   add_index "test_results", ["site_id"], name: "index_test_results_on_site_id", using: :btree
   add_index "test_results", ["uuid"], name: "index_test_results_on_uuid", using: :btree
 
+  create_table "transfer_packages", force: :cascade do |t|
+    t.integer "receiver_institution_id", limit: 4,                   null: false
+    t.string  "uuid",                    limit: 36,                  null: false
+    t.string  "recipient",               limit: 255
+    t.boolean "includes_qc_info",                    default: false
+  end
```

Comment:
Is there a particular reason for omitting timestamps on this model? We have them pretty much everywhere else and it's good practice for some limited keeping track of changes.

Reply:
Yes: I forgot :face_exhaling: 

I used `rails g migration` instead of `rails g model` so I ended with an empty migration file, instead of a basic one with `create_table`.

Reply:
And would be super useful. Let's add a migration to add them and push a patch release quickly?